test/commands/start.spec.ts

```diff
@@ -209,7 +209,7 @@ describe.skip('start', () => {
 	describe('when --api-ipc is specified', () => {
 		it('should update the config value', async () => {
 			await StartCommand.run(['--api-ipc'], config);
-			const [, usedConfig] = (application.getApplication as jest.Mock).mock.calls[0];
+			const [usedConfig,] = (application.getApplication as jest.Mock).mock.calls[0];
```

Comment:
Optional: the trailing comma seems to be unnecessary.
```suggestion
			const [usedConfig] = (application.getApplication as jest.Mock).mock.calls[0];
```